ajax/libs/algebra.js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "algebra.js",
+  "filename": "algebra.min.js",
+  "version": "0.2.6",
+  "description": "Build, display, and solve algebraic equations.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nicolewhite/algebra.js.git"
+  },
+  "keywords": [
+    "algebra",
+    "math",
+    "fractions",
+    "equations",
+    "expressions",
+    "latex"
+  ],
+  "author": "Nicole White",
+  "license": "MIT",
+  "homepage": "http://algebra.js.org",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nicolewhite/algebra.js.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
```

Comment:
https://github.com/nicolewhite/algebra.js/tree/master/build
Seems like all filenames including version number and all versions would be fetched for now. Maybe don't set up auto-update config for now is a better idea?

Reply:
As this discussion https://github.com/cdnjs/cdnjs/pull/8152#issuecomment-295075986 I think it's fine here.
What do you think?
Thanks.

Reply:
OK